File: openapi-client/php-dt/src/App/DTO/ErrorMessage.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * The response body returned when an error occurs.
 */
class ErrorMessage
{
    /**
     * The HTTP-style status code associated with the error.
     */
    #[DTA\Data(field: "code")]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $code = null;

    /**
     * A human-readable description of the error.
     */
    #[DTA\Data(field: "message")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $message = null;

}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/ErrorMessage.scala

package org.openapitools.client.model


case class ErrorMessage (
    /* The HTTP-style status code associated with the error. */
    _code: Integer,
    /* A human-readable description of the error. */
    _message: String
)
object ErrorMessage {
    def toStringBody(var_code: Object, var_message: Object) =
        s"""
        | {
        | "code":$var_code,"message":$var_message
        | }
        """.stripMargin
}
